rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -252,6 +303,8 @@ Once it's ready, transition the project to the next phase of its life-cycle. You
 
 === Transition to a Previous Phase
 
+If you need to rework an automation project, transfer it to a previous phase. All elements you configured are saved for when you come back.
```

Comment:
Same question about "for when you come back". 

Something like:
RPA retains your previously configured elements from each phase.